virt/getting_started/virt-web-console-overview.adoc

```diff
@@ -252,6 +252,11 @@ Preview features are disabled by default and must not be enabled in production e
 |*Live Migration* -> *Live migration network* list
 |Select a dedicated secondary network for live migration.
 
+|*SSH Configuration* -> *SSH over LoadBalancer service* toggle
+|Enable the creation of LoadBalancer services for SSH connections to VMs. 
+
+You will need cluster adminstrator permissions and a load balancer must be configured.
```

Comment:
Re: "will". Use present tense. However, in this case you do not need "You will need cluster administrator permissions" because all these settings require cluster admin. If someone without admin permissions logs in, these settings are disabled. So no need to state the requirement. 

Suggestion: "You must configure a load balancer."